integration/dkg.go

```diff
@@ -0,0 +1,152 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"testing"
+	"time"
+
+	ptypes "github.com/dedis/d-voting/proxy/types"
+	"github.com/dedis/d-voting/services/dkg"
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/dela/core/txn"
+	"go.dedis.ch/kyber/v3"
+	"golang.org/x/xerrors"
+)
+
+func initDkg(nodes []dVotingCosiDela, formID []byte, m txn.Manager) (dkg.Actor, error) {
```

Comment:
Again, all the functions need a little documentation. For someone who hasn't written the code, it's can be hard to understand how they're different and what is the purpose of each one